src/components/profile/profile.spec.tsx

```diff
@@ -37,40 +44,346 @@ describe("Profile", () => {
 
     describe("initials", () => {
       it("calculates the initials when not provided", () => {
-        instance = shallow(<Profile name="Foo Bar Baz" email="foo@bar.com" />);
+        instance = shallow(<Profile name="John Doe" email="john@doe.com" />);
         expect(instance.find(ProfileAvatarStyle).props().initials).toEqual(
-          "FBB"
+          "JD"
         );
       });
 
-      it("returns empty string when name is an empty string", () => {
-        instance = shallow(<Profile name="" email="foo@bar.com" />);
+      it("returns an empty string if no name is found", () => {
+        instance = shallow(<Profile name="" />);
         expect(instance.find(ProfileAvatarStyle).props().initials).toEqual("");
       });
 
-      it("returns empty string when name is a space", () => {
-        instance = shallow(<Profile name=" " email="foo@bar.com" />);
-        expect(instance.find(ProfileAvatarStyle).props().initials).toEqual("");
+      test("acronymize function generates correct acronym", () => {
```

Comment:
**comment**: I'm  not sure I like explicitly testing the `acronymise` function individually, particularly when that means exporting it from the component file. I see it as more of an implementation detail. Can we not test the same thing by just checking the text content of the appropriate element when various `name` values are provided as prop?